tests/Unit/ConfigProviderTest.php

```diff
@@ -14,9 +14,13 @@ class ConfigProviderTest extends TestCase
      */
     public function gettersAreValid(): void
     {
-        $configuration = new ConfigProvider();
+        $configuration = new ConfigProvider('test', 'Qwerty123!');
 
         $this->assertSame('http://ee.econt.com/', $configuration->getUrl());
         $this->assertSame('/services/Nomenclatures/', $configuration->getServiceURI());
+        $this->assertSame([
```

Comment:
formatting looks off, the array should not be on same line as `(`

Reply:
fixed